compatibilityTests/src/test/scala/org/apache/spark/sql/SnappyTableScanSuite.scala

```diff
@@ -19,91 +19,5 @@ package org.apache.spark.sql
 import org.apache.spark.sql.sources.TableScanSuite
 import org.apache.spark.sql.test.{SharedSnappySessionContext, SnappySparkTestUtil}
 
-class SnappyTableScanSuite
-  extends TableScanSuite
-    with SharedSnappySessionContext
-    with SnappySparkTestUtil {
-
-  // all tests in TableScanSuite will run except for below tests
-  // read comments above each test for desc on fixes
-  override def excluded: Seq[String] = Seq(
-    "exceptions",
-    "read the data source tables that do not extend SchemaRelationProvider",
-    "SPARK-5196 schema field with comment"
-  )
-
-  override def ignored: Seq[String] = Seq(
-    "Schema and all fields"
-  )
-
-  // SimpleScanSource is external to snappy. changed 'TABLE' to 'EXTERNAL TABLE'
```

Comment:
The semantics of "CREATE TABLE" have been relaxed like in Spark where both EXTERNAL tables and inbuilt table types (column/row/sample/...) are allowed. However "CREATE EXTERNAL TABLE" will fail for inbuilt table types. As per documentation we should still recommend using "CREATE TABLE" for inbuilt table types and "CREATE EXTERNAL TABLE" otherwise.